### PR TITLE
Added ztor estimation to Campbell and Bozorgnia (2014)

### PR DESCRIPTION
**Pull request by micheles**
Supersedes https://github.com/gem/oq-engine/pull/7744. Users of the engine can enable the new ztor formula by passing the parameter `better_ztor` to the GMPE (a better name is in order ;-)).

### CODE REVIEW

**Review by mmpagani — APPROVED** [review]

**Review by mmpagani — APPROVED** [review]

File: openquake/hazardlib/gsim/campbell_bozorgnia_2014.py [hunk 1]
```diff
@@ -356,6 +356,21 @@ class CampbellBozorgnia2014(GMPE):
...
     SJ = 0  # 1 for Japan
 
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+        self.better_ztor = int(kwargs.get('better_ztor', 0))
+
+    def set_parameters(self, ctx):
+        """
+        Use the ztor formula in equation 4 and 5 of Chiou & Youngs 2014
+        is the attribute .better_ztor is set.
```
[the remark this answers is not in the document]
Comment (micheles) [reply openquake/hazardlib/gsim/campbell_bozorgnia_2014.py]: Fixed

**Comment by micheles** [comment]
@SnowNooDLe, see if you like this approach.

**Comment by SnowNooDLe** [comment]
Oops, forgot to left a comment to ask.

So, I believe we are happy with the current approach? Hence, we can also implement `width` and `hypo_depth` in the same approach.
Like. `if self.estimate_width: do something`

Also, while we implement the rest two estimations, is it okay for us to rename `better_ztor` to `estimate_ztor`? and maybe type to Boolean which makes context, if we estimate, go ahead, estimate Ztor, if not, do not estimate! :)

**Comment by micheles** [comment]
Yes you can do all that. Please remember to change the corresponding line in the test case_71: https://github.com/gem/oq-engine/blob/master/openquake/qa_tests_data/classical/case_71/gmpe_logic_tree.xml#L16
The beauty of this approach is that now all users can use the estimate parameters, in regular engine calculations, if they wish, without having to resort to a custom function `get_mean_stds`.

**Pull request merged**